app/assets/stylesheets/bootstrap-overrides.scss

```diff
@@ -1,3 +1,7 @@
+body {
+  background-color: $sul-body-bg;
```

Comment:
Doesn't `$body-bg` handle this? https://github.com/twbs/bootstrap/blob/ba5b5a630081fae0e793218ad49364ba46d15605/scss/_reboot.scss#L57 Or is this preventing dark-mode?

Reply:
`body-bg` also gets used for cards and all sorts of things, and I guess we want those backgrounds white but the true body bg to be that beige (for now.. I think that's changing in 4.0) 🤷‍♂️